tests/unit/stream_alert_rule_processor/test_handler.py

```diff
@@ -346,7 +346,7 @@ def test_do_not_invoke_threat_intel(self, load_intelligence_mock):
         self.__sa_handler = StreamAlert(get_mock_context(), False)
         load_intelligence_mock.assert_called()
 
-    def test_firehose_sanitize_keys(self):
+    def test_firehosesanitize_keys(self):
```

Comment:
unintentional removal of `_`?

Reply:
nope that was just a side  effect of find & replace